tests/e2e/settings-profile.test.ts

```diff
@@ -64,7 +64,7 @@ test('Users can update their profile photo', async ({ login, page }) => {
 	await expect(page).toHaveURL(`/settings/profile/photo`)
 
 	await page
-		.getByLabel(/change/i)
```

Comment:
This is flaky and results in "node is not an htmlinputelement", guess file input type 
 nodes are not treated the same as text input nodes.